cudax/examples/stf/word_count_reduce.cu

```diff
@@ -0,0 +1,72 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of CUDASTF in CUDA C++ Core Libraries,
+// under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+// SPDX-FileCopyrightText: Copyright (c) 2022-2024 NVIDIA CORPORATION & AFFILIATES.
+//
+//===----------------------------------------------------------------------===//
+
+/**
+ * @file
+ * @brief Counting words in a text using a launch kernel
+ */
+
+#include <cuda/experimental/stf.cuh>
+
+using namespace cuda::experimental::stf;
+
+// determines whether the character is alphabetical
+__host__ __device__ bool is_alpha(const char c)
+{
+  return (c >= 'A' && c <= 'z');
```

Comment:
Actually that's incorrect because there are a few nonalpha characters between `Z` and `a`. 

```suggestion
  return c >= 'A' && c <= 'Z' || c >= 'a' && c <= 'z';
```

Reply:
I will check what the Thrust example does …

Reply:
Whoa chatgpt just suggested this clever thing:

```suggestion
  return ((c | 0x20) - 'a') < 26;
```

Reply:
Fixed ...